R/data/data_input_panel.R

```diff
@@ -153,11 +153,42 @@ data_input_panel_server <- function(id, continuous_file = 'data/Non_opioids_long
         invalid_data(NULL)
       }
       
+      result = ValidateUploadedData(df, input$metaoutcome)
+
+      if (!result$valid) {
+        showModal(
+          modalDialog(
+            title = "Invalid Data",
+            easyClose = TRUE,
+            p("Uploaded data was invalid because:"),
+            p(tags$strong(result$message)),
+            p("Please check you data file and ensure that you have the correct outcome type selected on the home page"),
+            shinyBS::bsCollapse(
+              shinyBS::bsCollapsePanel(
+                title = "Show Data",
+                div(
+                  style = 'overflow-x: scroll',
+                  DT::dataTableOutput(outputId = ns("invalid_data"))
+                )
+              )
+            ),
+            modalButton(label = "OK"),
+            footer = NULL
+          )
+        )
+        invalid_data(df)
+        output$file_input_panel <- default_file_input
+        df <- defaultD()
+        data_uploaded(FALSE)
+      } else {
+        invalid_data(NULL)
+      }
+      
```

Comment:
This section was removed on dev. It was removed to correct a merge error on my part.

Reply:
Done :)